NAMESPACE

```diff
@@ -82,13 +85,16 @@ importFrom(Seurat,ScaleData)
 importFrom(Seurat,VariableFeatures)
 importFrom(Seurat,as.SingleCellExperiment)
 importFrom(SingleR,SingleR)
+importFrom(SummarizedExperiment,"rowData<-")
 importFrom(SummarizedExperiment,SummarizedExperiment)
 importFrom(SummarizedExperiment,assays)
 importFrom(SummarizedExperiment,colData)
 importFrom(SummarizedExperiment,rowData)
 importFrom(callr,r)
 importFrom(celldex,BlueprintEncodeData)
 importFrom(celldex,MonacoImmuneData)
+importFrom(cellxgene.census,get_seurat)
+importFrom(cellxgene.census,get_single_cell_experiment)
```

Comment:
We should not import from census. HPCell is database agnostic.

What is the reason we need census? (this is a separate framework to CuratedAtlasQuery)